static/app/views/settings/featureFlags/organizationFeatureFlagsProviderRow.tsx

```diff
@@ -10,8 +10,8 @@ import {Tooltip} from 'sentry/components/tooltip';
 import {IconSubtract} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
+import useUserFromId from 'sentry/utils/useUserFromId';
```

Comment:
We could reuse the hook introduced as a part of `featureFlags` code, so I have moved it to the shared hooks directory and replaced just the location of the import here.